### ISSUE TITLE
Error on 'Reset Password' clicked 

### ISSUE THREAD

**Issue by SmartPlugins** [okta/okta-signin-widget#2903]
### Describe the bug

On click of the 'Forgot Password' link, there is an exception/error triggered 
"No remediation can match current flow, check policy settings in your org. Remediations: [identify-recovery]" .

The issue seems to be happening only in the Interaction Code flow (OIE), when Interaction Code flow is turned off the Password reset seems to be working fine.

### What is expected to happen?

Expected to see the Forgot Password form

### What is the actual behavior?

"No remediation can match current flow, check policy settings in your org. Remediations: [identify-recovery]" 

### Reproduction Steps

I may be missing some OIE setting for the Self Service Password Reset (SSPR) flow under the interaction code flow.

### SDK Versions

We tried all these versions of the Okta Signin widget	
v6.4.1, v6.6.2, v7.0.0,

### Execution Environment

Chrome

**Comment by oleksandrpravosudko-okta** [comment]
Thanks for report @SmartPlugins.
Are you specifying [`flow`](https://github.com/okta/okta-signin-widget#flow) option when initializing widget?
In case you have not checked it already, you could verify if your org's configuration matches this guide:
https://help.okta.com/oie/en-us/Content/Topics/identity-engine/authenticators/configure-sspr.htm

**Comment by SmartPlugins** [comment]
Yes we are setting the flow option and the SSPR settings match the link provided,

**Comment by MurthyAvanithsa** [comment]
We have the same issue, this happens especially when we try providing the code sent in the reset password ink

**Comment by master12** [comment]
We have a similar error. If we specify flow something other than default or resetPassword we will get this error.

For example if we set flow to 'signup' we will get this error if someone tries to go to the forgot password link

**Comment by jaredperreault-okta** [comment]
@master12 Do you have profile enrollment enabled on your org? https://help.okta.com/oie/en-us/Content/Topics/identity-engine/policies/about-ssr-and-pe.htm

**Comment by jaredperreault-okta** [comment]
If this problem persists you can try reaching out to https://support.okta.com/. They can help with setting up your org configuration to enable these flows

**Comment by denysoblohin-okta** [comment]
Internal ref: OKTA-585074
